doc/getting-started/install.md

```diff
@@ -74,25 +75,19 @@ Confirm that you installed cabal version `3.6.2.0`:
 
 Download and install version `8.10.7` of GHC.  The easiest way to do this is to use [ghcup](https://www.haskell.org/ghcup/).
 
-```bash
-$ curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh
-```
+    curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh
```

Comment:
no more ```bash?

Reply:
I prefer using `bash`, but most of the file doesn't use it.  Alternative, we could modify the file to use `bash` everywhere.

Reply:
I pushed a version that switches to `bash`.  If this is good, then I'll squash.  Otherwise, I'll revert then squash.